activities/PhysicsJS.activity/package.json

```diff
@@ -6,5 +6,9 @@
       "sugar-web": "github:sugarlabs/sugar-web/master",
       "domReady": "github:requirejs/domReady/2.0.1"
     }
+  },
+  "dependencies": {
+    "fs": "^0.0.1-security",
+    "js-ini": "^1.6.0"
```

Comment:
Not sure to understand why you add this.

Reply:
@llaske resolved that part. There is no need to mention that part. Removed it & updated in commit [205510c](https://github.com/llaske/sugarizer/pull/1478/commits/205510c3e4b4e714907b482d62f6494f5ba0161d).
Still it works properly attaching ss 
![Screenshot 2024-01-12 091712](https://github.com/llaske/sugarizer/assets/144143093/2f8f9855-6f30-4d7e-b01a-354d0049e789)